src/main/scala/devices/debug/Debug.scala

```diff
@@ -603,12 +611,14 @@ class TLDebugModuleInner(device: Device, getNComponents: () => Int, beatBytes: I
     // Registers coming from 'CONTROL' in Outer
     //--------------------------------------------------------------
 
-    val selectedHartReg = RegInit(0.U(10.W))
+    val selectedHartReg = RegInit(0.U(p(MaxHartIdBits).W))
```

Comment:
do we need a `require p(MaxHartIDBits <= 10)`?

Reply:
The regmapper will catch it, because it'll cause the reg fields to overlap in the address map.